charts/shoot-core/charts/csi-packet/templates/setup.yaml

```diff
@@ -0,0 +1,132 @@
+apiVersion: storage.k8s.io/v1beta1
```

Comment:
Why is this here? Shouldn't it be created via PacketBotanist, same like https://github.com/gardener/gardener/blob/master/pkg/operation/cloudbotanist/alicloudbotanist/addons.go#L48?

Reply:
Yes, and I did add it in the go code, but only after doing the yml, so didn't realize it was there. Removed.